GoogleForCreators/web-stories-wp
# Block: carousel layout responsiveness issues

## Context

Most notably on Firefox on smaller screens, the Glider track width calculation is quite off, causing white space to appear at the beginning of the carousel.

Noting per https://github.com/GoogleForCreators/web-stories-wp/issues/12170#issuecomment-1294864619 I was able to re-create in Chrome as well.



## Summary
Per https://github.com/GoogleForCreators/web-stories-wp/issues/12170#issuecomment-1294868604 

Removes justify content center.  `margin: 0 auto;` should continue to handle the centering.

See: https://www.youtube.com/watch?v=dCpdV3PurC0



## Relevant Technical Choices



## To-do



## User-facing changes



## Testing Instructions






This PR can be tested by following these steps:

1. Add Web Stories block with carousel layout, make sure there are plenty of stories in it
2. View in Firefox w/ small viewport (+ large ones to ensure backwards compat)
3. Previous gap should be fixed


## Reviews

### Does this PR have a security-related impact?

No



### Does this PR change what data or activity we track or use?

No



### Does this PR have a legal-related impact?

No



## Checklist




---



Fixes #12170
